### ISSUE TITLE
Docs for new comers: Calling side of django-watchman

### ISSUE THREAD

**Issue by guettli** [mwarkentin/django-watchman#97]
I think it would help new comers, if you could tell some words about the "calling side" of django-watchman. I mean the client which calls the http end point and which collects and acts on the reported status messages.

I know that this is not the job of django-watchman, but it helps new comers to understand the big picture.

**Comment by mwarkentin** [comment]
@guettli Thanks for the suggestion! I agree that this would be really helpful - it would be nice to have configuration instructions for pingdom, panopta, and so on.

If I have some time I'll try to add some documentation, and of course contributions are always welcome!

**Comment by guettli** [comment]
I know that new features look far more appealing than new comer friendly docs. Nevertheless I still think that new comer friendly docs containing a "big picture" are the key to success.

**Comment by mwarkentin** [comment]
@guettli I have to agree with you there - I've made sure this issue won't be closed as stale. Any chance you're interested in helping to get that sort of doc started? Even a rough outline of what you think the main pieces might be would be helpful.

**Comment by guettli** [comment]
@mwarkentin thank you for looking at this issue again. Up to now I don't use django-watchman since we have not decided which tool we want to use to collect the results. That's why I am curious what you suggest :-)

**Comment by mwarkentin** [comment]
Ok makes sense. Personally I’ve used both Pingdom and more recently Panopta. Both have worked well for us. 

Seems like it would make sense to have an “integrations” section to the doc where it could walk through the configuration for various services.

**Comment by mwarkentin** [comment]
Some integrations we could document:

* https://upptime.js.org
* https://uptimerobot.com
* https://www.statuscake.com
* https://www.pingdom.com
* https://www.panopta.com
* https://updown.io
* https://cronitor.io
* https://betteruptime.com
